tests/testcases/a_basics/zdb_testcases.py

```diff
@@ -157,4 +157,58 @@ def test004_start_stop_zerodb(self):
         namespaces = zdb.namespace_list()
         self.assertEqual(len(namespaces.result), 1)
         self.assertEqual(namespaces.result[0]['name'], ns_name)
-        self.zdbs.append(zdb)
\ No newline at end of file
+        self.zdbs.append(zdb)
+
+    def test005_delete_namespaces(self):
+        """ ZRT-ZOS-022
+        *Test case for deleting namespaces*
+
+        **Test Scenario:**
+
+        #. Create zerodb (zdb) with namespace, should succeed.
+        #. List namespaces, NS should be found.
+        #. Delete namespace (NS), should succeed. 
+        #. List namesapces, should be empty. 
+        """
+        self.log('Create zerodb (zdb) with basic params, should succeed')
+        ns_name = self.random_string()
+        namespace = [{'name': ns_name, 'size': random.randint(1, 30), 'password': self.random_string(), 'public': False}]
+        zdb = self.controller.zdb_manager
+        zdb.install(wait=True, path=self.mount_paths, namespaces=namespace)
+        self.zdbs.append(zdb)
+
+        self.log('List namespaces, NS should be found')
+        namespaces = zdb.namespace_list()
+        self.assertEqual(len(namespaces.result), 1)
+        self.assertEqual(namespaces.result[0]['name'], ns_name)
+        
+        self.log("Delete namespace (NS), should succeed.")
+        delete = zdb.namespace_delete(name=ns_name)
+        time.sleep(2)
+        self.assertEqual(delete.state, 'ok')
+
+        self.log('list the namespaces, should be empty')
+        namespaces = zdb.namespace_list()
+        self.assertEqual(namespaces.result, [])
```

Comment:
i think u also need to make sure from the node itself that it is actually deleted